kibana_prometheus_exporter/kibana_collector.py

```diff
@@ -156,7 +156,7 @@ def __init__(
         path: str = "/api/status",
         kibana_login: str = None,
         kibana_password: str = None,
-        ignore_ssl: str = None,
+        ignore_ssl: bool = None,
```

Comment:
This should read `ingore_ssl: bool = False` because the default is not "undefined" but "don't ignore ssl".
Technically `None` behaves as expected because `not None == True`, but it's incorrect.

Reply:
I see, just changed it